software/kernel/boards/ambient_module/Makefile

```diff
@@ -1,12 +1,10 @@
-# Makefile for building the tock kernel for the Hail platform
```

Comment:
Is there any real benefit to this comment? Better to just delete it (and its brethren) rather than copying them around everywhere? [It's obviously a Makefile, and it's in the folder it's going to build for...]